app/src/main/java/org/oppia/app/hintsandsolution/HintsAndSolutionAdapter.kt

```diff
@@ -188,7 +193,9 @@ class HintsAndSolutionAdapter(
               position
             }
           expandedHintListIndexListener.onExpandListIconClicked(currentExpandedHintListIndex)
-          if (previousIndex != null && currentExpandedHintListIndex != null && previousIndex == currentExpandedHintListIndex) {
+          if (previousIndex != null && currentExpandedHintListIndex != null &&
+            previousIndex == currentExpandedHintListIndex
```

Comment:
Ditto

Reply:
ktlint says line cannot start with `&&`
`Line must not begin with "&&"`

Reply:
Same comment as above

Reply:
Done